src/inet/routing/leach/Leach.ned

```diff
@@ -0,0 +1,38 @@
+//
+// SPDX-License-Identifier: LGPL-3.0-or-later
+//
+
+package inet.routing.leach;
+
+import inet.routing.contract.IManetRouting;
+
+//
+// This module provides the sensor implementation of 
+// Low-Energy Adaptive Clustering Hierarchy (LEACH) routing, a hierachical 
+// adhoc routing protocol for Wireless Sensor Networks. 
+// This implementation is designed to use a a single wireless interface and run on
+// the IEEE 802.15.4 standard for MAC and Physical layers.
+// Therefore the standard TDMA scheduling system has been implemented 
+// with delayed time slots due to the use of CSMA in the 802.15.4 standard.
+//
+// For more information on the algorithm, see the LEACH paper
+// https://ieeexplore.ieee.org/document/926982
+//
+simple Leach like IManetRouting
+{
+    parameters:
+        // properties
+        @class("leach::Leach");
+        @display("i=block/routing");
+        
+        // context parameters
+        string interfaceTableModule; 
+        
+        // LEACH parameter group
+        double maxVariance = default(1);
+        int numNodes = default(10);
+        double clusterHeadPercentage = default(0.5);
```

Comment:
Missing documentation and non descriptive / misleading name. Generally, if you specify a default value in a NED file, then that value must be supported by the specification document somewhere. If this is just a random picked value, then do NOT set it as a default value, but force the user to set it explicitly in the INI file. Besides the name is suspicious. You meant half a percent as a default value here? or rather it should be named `clusterHeadRatio` and then the value represents 50% ?

recommendation: `double clusterHeadRatio @unit(ratio) = default(0.5ratio)` and in that case, you can even use = 50pct because omnet can convert automatically between ratio and pct

Reply:
> Generally, if you specify a default value in a NED file, then that value must be supported by the specification document somewhere.

Noted. 
The original paper authors have provided 0.05 (or 5%) as the **desired percentage of cluster heads** for their experiments so took your recommendation and modified the code in https://github.com/inet-framework/inet/pull/929/commits/f04b5be512cb34a55b92b7eacf6856c6e85ddc0d